integration/mediation-tests/tests-mediator-1/src/test/resources/artifacts/ESB/mediatorconfig/call/AcceptedEndpointProxy.xml

```diff
@@ -0,0 +1,25 @@
+<definitions xmlns="http://ws.apache.org/ns/synapse">
```

Comment:
Shall we add the license header to all the files that are missing it?

Reply:
Fixed in https://github.com/wso2/product-ei/pull/4779